tests/radio.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { test, expect } from '@playwright/test';
+import { waitForCustomEvent, waitForCustomEventTimeout } from './lib/helper.ts'
+
+
+test('Radio E2E Test, Options, Selected, Click, Keydown ', async ({ page }) => {
```

Comment:
i think it's generally good practice to name tests by the action they're testing (i.e. Renders options appropriately, Responds to click, etc), kinda like naming PRs. this makes it easier to debug when tests break as well. it's fine to omit the component name because the tests are already named

a good example in the code is `listbox.spec.ts` -- lmk if you agree, we should probably standardize this moving forwards

Reply:
also would be good to split up these tests (options, selected, click, keydown) for the same debugging reasons

Reply:
Thanks, good point!  Will update my tests to match list box's naming convention + break tests up by action.  I now realize my tests are so boring...I wish I wrote pokemon themed tests now.  

I should probably go back and update the other PRIME tests I've written to also match this convention.

Reply:
me too... lol